docs/ldoc.ltp

```diff
@@ -197,6 +202,9 @@
 #   if item.display_type then
     <span class="proptype">($(item.display_type))</span>
 #   end
+#   if item.inherited then
+    <span class="baseclass" $(nowrap)>&nbsp;&middot;&nbsp;Inherited from $(item.baseclass)</span>
```

Comment:
Ah, I did the same thing on my side and added signals count, "introduced in" and theme variables count. It's my fault for not pushing that first :). I will play with some more ideas. Maybe a "tag cloud"/"gmail label style" or vertical separators. I will see. Anyhow, they will integrate fine since all of those things are "metadata".

Reply:
You're always pushing things to the next level! Can't wait to see it! :)